### PR TITLE
refactor(core): 'Paginate' is default to showPages={0}

### LINKED ISSUE #1405: Paginate should default to showPages={0}
See: https://github.com/mozilla/addons-frontend/pull/1398#discussion_r88934951

### PR DESCRIPTION
**Pull request by tinhtooaung**
Fixes #1405

### CODE REVIEW

**Comment by muffinresearch** [comment]
@tinhtooaung hi - do you have a bug reference for this patch?

**Comment by tinhtooaung** [comment]
hi @muffinresearch, the issue for this pull request is here: https://github.com/mozilla/addons-frontend/issues/1405

Please let me know if you need anything to change it. This is my first pull request :-D

File: src/core/components/Paginate/index.js [hunk 1]
```diff
@@ -51,10 +51,10 @@ export class PaginateBase extends React.Component {
     // If we can show all of the pages, show them all.
     if (pageCount <= showPages) {
       return makePageNumbers({ start: 1, end: pageCount });
-    // If we are showing less on the right than we should, define the start by the end.
+      // If we are showing less on the right than we should, define the start by the end.
```
Comment (tofumatt) [inline src/core/components/Paginate/index.js]: Not sure the reason for this indentation change; can you restore the comments to be indented as they were before?

File: src/core/components/Paginate/index.js [hunk 2]
```diff
@@ -108,7 +108,7 @@ export class PaginateBase extends React.Component {
           })}
           {this.visiblePages().map((page) => (
             this.makeLink({ currentPage, page, pathname, queryParams }
-          )))}
+            )))}
```
Comment (tofumatt) [inline src/core/components/Paginate/index.js]: This should not be indented more. (Shame our tests didn't catch it though)

File: src/core/components/Paginate/index.js [hunk 3]
```diff
@@ -51,10 +51,10 @@ export class PaginateBase extends React.Component {
     // If we can show all of the pages, show them all.
     if (pageCount <= showPages) {
       return makePageNumbers({ start: 1, end: pageCount });
-    // If we are showing less on the right than we should, define the start by the end.
+      // If we are showing less on the right than we should, define the start by the end.
     } else if (end - currentPage < showExtra) {
       return makePageNumbers({ start: (end - showPages) + 1, end });
-    // If we are showing less on the left than we should, define the end by the start.
+      // If we are showing less on the left than we should, define the end by the start.
```
Comment (tofumatt) [inline src/core/components/Paginate/index.js]: This indent too.

File: src/admin/components/SearchPage.js [hunk 4]
```diff
@@ -31,7 +31,7 @@ export default class AdminSearchPage extends React.Component {
     const pathname = '/search/';
     const paginator = hasSearchParams && count > 0 ?
       (<Paginate count={count} currentPage={page}
-        pathname={pathname}
+        pathname={pathname} pageSize={9}
```
Comment (tofumatt) [inline src/admin/components/SearchPage.js]: Should this be `showPages` not `pageSize`?

**Comment by tinhtooaung** [comment]
Hi @tofumatt I am on to it now. Not sure where those indents come from. Could be my text editor?
I will fix up.

**Comment by tinhtooaung** [comment]
Hi @tofumatt I just submitted an amended commit to this pull request. Following are the changes I did:

- Fix indentation per suggestions
- Fix pageSize to showPages
- Add a test verifying default showPages is 0

Please feel free to let me know if there are more things to be done :-)

**Comment by tofumatt** [comment]
Looks like [you've got some linting errors](https://travis-ci.org/mozilla/addons-frontend/jobs/193894521#L337), make sure to run `npm test` locally to catch these locally 😄

**Pull request closed without merging**